### DISCUSSION TITLE
FlexibleUNet performance. Benchmarked?

### DISCUSSION THREAD

**Discussion by ibro45** [Project-MONAI/MONAI#6562] [Q&A] [answered]
Hi, has FlexibleUNet been benchmarked? I have been trying to use it in two different cases, and I've been getting subpar performance in both. Specifically, I 1) tried it on tasks 6 and 9 of the Decathlon dataset, and 2) used it inside of the nnUNet framework in place of the default UNet to train on TotalSegmentator data. In 1) it performed just ok, definitely subpar, and in 2) it performed way below the default PlainConvUNet network from the nnUNet.

I used a 3D EfficientNet-B4 as backbone, and tried with and without 'deconv' upsampling.

**Comment by KumoLiu** [comment]
Hi @ibro45, I don't think we have benchmark results, maybe you can try to adjust the hyperparameter or go with some simpler network first.
@binliunls, could you please help double-confirm here, thanks!

**Reply by binliunls** [reply, marked as answer]
Hi @ibro45  , 
    No, we don't have benchmark results about 3D images yet. It was only benchmarked on the 2D endoscopic tool segmentation task currently. Here are some tips for reference if you are going to use the FlexibleUNet.
    1) Try a smaller backbones like efficientnet-b0 or efficientnet-b2. We found the FlexibleUNet with a large backbone is easy to overfit.
    2) Try to load an pre-trained efficientnet weight trained on a large classification task. As we tested, a pre-trained weight could help to get a better result.

Thanks,
Bin
